pkg/types/openstack/validation/machinepool.go

```diff
@@ -7,6 +7,26 @@ import (
 )
 
 // ValidateMachinePool checks that the specified machine pool is valid.
-func ValidateMachinePool(p *openstack.MachinePool, fldPath *field.Path) field.ErrorList {
-	return field.ErrorList{}
+func ValidateMachinePool(platform *openstack.Platform, p *openstack.MachinePool, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	// Validate Root Volumes
```

Comment:
I don't understand this change? Why are we setting when validating? That's not how we do it for other platforms...